README.md

```diff
@@ -16,30 +16,34 @@
 [![Coverage Status](https://coveralls.io/repos/github/spyder-ide/spyder/badge.svg?branch=master)](https://coveralls.io/github/spyder-ide/spyder?branch=master)
 [![codecov](https://codecov.io/gh/spyder-ide/spyder/branch/master/graph/badge.svg)](https://codecov.io/gh/spyder-ide/spyder)
 
-----
+![Screenshot of Spyder's main window](./img_src/screenshot.png)
 
-## Important Announcement: Spyder is unfunded!
+----
 
-Since mid November/2017, [Anaconda, Inc](https://www.anaconda.com/) has
-stopped funding Spyder development, after doing it for the past 18
-months. Because of that, development will focus from now on maintaining
-Spyder 3 at a much slower pace than before.
+## Important Announcement: Spyder needs your support!
 
-If you want to contribute to maintain Spyder, please consider donating at
+Since mid November 2017, [Anaconda, Inc](https://www.anaconda.com/) has
+stopped funding Spyder development, after doing so for the past 18
+months. Therefore, without additional funds, development will shift to
+maintaining Spyder 3 at a much slower pace than before, while slowly working
```

Comment:
We could be more positive and have the same impact by removing "much" and "slowly" from this line. 

I think it would be better for people to think things will still be moving forward(not dead) but could be faster with help.

Reply:
I absolutely agree and would have done so, but I didn't want to change it too much from @ccordoba12 's original wording without his approval. Opinion @ccordoba12 ?